src/index.html

```diff
@@ -8,7 +8,224 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <!--Header-->
+      <header>
+        <div>
+          <span>Home</span>
+          <span style="text-align: right;">Download My Full Resume</span>
+
+        </div>
+      </header>
+
+      <body>
+        <div id="top">
+          <h1 style="text-align: center">John Roberts</h1><br>
+          <h2 style="text-transform: uppercase;text-align: center">business expert</h2>
```

Comment:
Good to know the advanced css attribute, like `text-transform` and the usage of h1 and h2 is correct 👍 
But it's better to extract the inline styles into the external css file once we complete the CSS lessons.